### ISSUE TITLE
Invalid link : https://falco.org/docs/installation/#daemonset-manifests

### ISSUE THREAD

**Issue by nlamirault** [falcosecurity/falco#1214]
According to the documentation : 
Clone the Falco repo and change to the directory with the manifests.

```
git clone https://github.com/falcosecurity/falco/
cd falco/integrations/k8s-using-daemonset
```

But directory only exists on tag. We must checkout tags and not the master branch.

**Comment by leodido** [comment]
/kind documentation
L.

**Comment by leogr** [comment]
IIRC that page will be removed by https://github.com/falcosecurity/falco-website/pull/184

Anyhow, I believe we should add some reference to https://github.com/falcosecurity/contrib

**Comment by leodido** [comment]
@nlamirault I think what you're looking for is now at https://github.com/falcosecurity/contrib/tree/master/deploy/kubernetes

**Comment by leogr** [comment]
> @nlamirault I think what you're looking for is now at https://github.com/falcosecurity/contrib/tree/master/deploy/kubernetes

Yes, exactly. Btw, we may want to change the repository name (see https://github.com/falcosecurity/contrib/issues/14 ), thus I would wait until that.

Moreover, I'm transferring this issue to https://github.com/falcosecurity/falco-website
